README.md

```diff
@@ -1,12 +1,12 @@
-# microbit-svg
+### microbit-svg
```

Comment:
Not sure why this title were changed, it is fine at the `h1` format (single `#`)
```suggestion
# microbit-svg
```